### PR TITLE
Update UG/DG

### PR DESCRIPTION
**Pull request by w-yuchen**
Updated UG for find.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -301,20 +301,25 @@ _{Explain here how the data archiving feature will be implemented}_
...
+| `* *`    | user                                       | update the status of property listings(completed, Option, Sale Agreement)  | follow up with clients on time                                                            |
+| `* *`    | user                                       | edit the deadline of the property listing                                  | extend the time period for selling the property                                           |
+| `* *`    | user                                       | edit the name of the property listing                                      | change the name of the property when I have made a mistake                                |
+| `* *`    | user                                       | edit the address of the property listing                                   | change the address of the property when I have made a mistake                             |
+| `* *`    | user                                       | edit the address of the property listing                                   | change the address of the property when I have made a mistake                             |
```
Comment (dvdweien) [inline docs/DeveloperGuide.md:322]: may i know whats changed here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -185,35 +185,62 @@ Examples:
...
-* `find property [keywords] [option...]`
+* `find property [KEYWORD] [OPTION...]`
 
 Description:
-* There can be 0 or more keywords and 0 or more options, but keywords and options cannot be both empty.
+* There can be 0 or more keywords and 0 or more options, but keywords and options cannot be both empty. All text are case insensitive. 
 
 Options:
-* `r/[REMARKS]`
+* `t/[TYPE OF HOUSING]`
```
Comment (w2vgd) [inline docs/UserGuide.md]: I think we should be consistent and use `t/PROPERTY_TYPE` here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -185,35 +185,62 @@ Examples:
...
 
-Search for properties or appointments whose remarks field contain patterns specified in `[REMARKS]`
+    Search for properties or appointments whose housing type field contain patterns specified in `[REMARKS]`.
```
Comment (w2vgd) [inline docs/UserGuide.md]: Need to remove appointments here

File: docs/UserGuide.md [hunk 4]
```diff
@@ -185,35 +185,62 @@ Examples:
...
-* `find property "jurong west"`
-* `find appointment "fri" r/"come in afternoon"`
+* `find property jurong west`
 * `find property pm/500000`
-* `find property "bishan" "north" "mrt" r/"recently renovated"`
+* `find property bishan north t/hdb pl/$1,000,000`
+
+### Searching appointments: `find appointment`
+
+Finds properties that match the criterion provided.
```
Comment (w2vgd) [inline docs/UserGuide.md]: This should be `appointments` instead

File: docs/UserGuide.md [hunk 5]
```diff
@@ -185,35 +185,62 @@ Examples:
...
-### Filtering: `find`
+### Searching properties: `find property`
 
-Finds properties or appointments that matches the criterion provided.
+Finds properties that match the criterion provided.
 
 Formats:
-* `find appointment [keywords] [option...]`
-* `find property [keywords] [option...]`
+* `find property [KEYWORD] [OPTION...]`
```
Comment (w2vgd) [inline docs/UserGuide.md:201]: I think it will be better if the `...` are after the square brackets.
E.g. `find property [KEYWORD] [OPTION]...`

**Review by w2vgd — COMMENTED** [review]
Some comments

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (w-yuchen) [reply docs/DeveloperGuide.md:322]: added 313 to 315

**Review by w2vgd — APPROVED** [review]
LGTM

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**